rlclientlib/utility/context_helper.cc

```diff
@@ -41,4 +44,64 @@ namespace reinforcement_learning { namespace utility {
       RETURN_ERROR_LS(trace, status, json_parse_error) << error_code::unknown_s;
     }
   }
+
+  int get_event_ids(std::map<size_t, std::string>& event_ids, const char *context, i_trace* trace, api_status* status) {

```

Comment:
Can we move output (and non-const) variables after input (and const) ones? It improves readability.